TestCases/species_transport/venturi_primitive_3species/species2_primitiveVenturi_mixingmodel.cfg

```
@@ -0,0 +1,139 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Species mixing with 2 species, i.e. 1 transport equations  %
+% Author: T. Kattmann                                                          %
+% Institution: Bosch Thermotechniek B.V.                                       %
+% Date: 2021/10/14                                                             %
```

Seems like a small thing that git blame would quickly reveal, but when you want to push into develop this work (e.g. testcase) is your responsibility and using other people credentials with old timestamps on it is not the way to go imo. Especially because you modified the cfg which this T. Kattmann dude pushed back in the day. By no means am I a copyright expert but I prefer just updating to your own credentials if it is some real addition and not just some  cleanup of the cfg.
```suggestion
% Case description: Species mixing with 2 species, i.e. 1 transport equations  %
%                    Including mixture dependent density                       %
% Author: Cristopher Morales Ubal                                              %
% Institution: Eindhoven University of Technology                              %
% Date: 2022/06/15                                                             %
```